test/support/stubs/OrderStub.js

```diff
@@ -0,0 +1,61 @@
+import constants from '../../../src/constants';
+
+const OrderStub = {
```

Comment:
took an OrderStub out into it's own file...it's called this is a stub, and not a mock, right?